src/metabase/api/dashboard.clj

```diff
@@ -88,26 +107,36 @@
                    (not= embedding_params (:embedding_params dash))))
       (check-embedding-enabled)
       (check-superuser)))
-  (check-500 (-> (assoc dashboard :id id)
-                 (dashboard/update-dashboard! *current-user-id*))))
-
-
+  (check-500
+   (db/update! Dashboard id
+     ;; description, position are allowed to be `nil`. Everything else must be non-nil
+     (u/select-keys-when dashboard
+       :present #{:description :position}
```

Comment:
Just for making it clear to myself: what are the scenarios when the value of position is `nil`?